src/olympia/reviewers/views.py

```diff
@@ -851,8 +851,10 @@ def review(request, addon, channel=None):
     actions_comments = [k for (k, a) in actions if a.get('comments', True)]
 
     deleted_addon_ids = (
-        ReusedGUID.objects.filter(guid=addon.guid).values_list(
-            'addon_id', flat=True) if addon.guid else [])
+        AddonGUID.objects.filter(guid=addon.guid)
+                         .exclude(addon=addon)
```

Comment:
Do we need this `exclude` now because we don't want the `id` of the current `addon` included in this list? Is there any test coverage for this change? I.e., should there be a test where the `id` for the current `addon` *is* in `AddonGUID`? Maybe that's already covered in the changes to the tests?

Reply:
> Do we need this `exclude` now because we don't want the `id` of the current `addon` included in this list? 

Yes.  The current addon wouldn't have been in ReusedGUID before, but _is_ in AddonGUID now.

> Is there any test coverage for this change?

the existing test lines below would fail if we don't exclude the current add-on
```
expected = [
            (f'{old_one.id}', reverse('reviewers.review', args=[old_one.id])),
            (f'{old_two.id}', reverse('reviewers.review', args=[old_two.id])),
        ]
```